source/wp-content/themes/wporg-news-2021/block-template-parts/front-page/latest-posts.html

```diff
@@ -1,7 +1,7 @@
 <!-- `perPage` must match the value in `offset_paged_posts()`. -->
 <!-- wp:query {"query":{"perPage":5},"tagName":"section","className":"front__latest-posts","align":"full"} -->
 <section class="front__latest-posts alignfull">
-	<!-- wp:heading {"level":2,"className":"front__latest-posts-heading"} -->
+	<!-- wp:heading {"level":2,"className":"front__latest-posts-heading front__section-heading"} -->
```

Comment:
can you remove the classes from all the `className`s too?

Reply:
Blerg. I must have had some brain fog yesterday. a1df4a8